### ISSUE TITLE
Nuclear Repulsion Energy Parsing for Gaussian Logs

### ISSUE THREAD

**Issue by migatt** [cclib/cclib#1241]
Does cclib provide a method for parsing the nuclear repulsion energy from gaussian logfiles?

I did not find anything in the documentation, but PR #142.

**Comment by berquist** [comment]
Yes, the code is located at https://github.com/cclib/cclib/blob/9020b6a839e43573dc5490d976a36dc88907d6ec/cclib/method/nuclear.py#L103 with an example of how to use it at https://github.com/cclib/cclib/blob/9020b6a839e43573dc5490d976a36dc88907d6ec/test/method/testnuclear.py#L70.

I'll make a note that it could be added to docs.  Is this enough to get started with?

**Comment by migatt** [comment]
Thank you very much for your quick and comprehensive answer!

Just to be sure, about which unit are we talking here?

**Comment by shivupa** [comment]
According to the discussion in #1242, The units in the NRE calculation need to be converted to eV. This issue cannot be closed until this is addressed.
